spec/acceptance/self_service_spec.rb

```diff
@@ -156,6 +156,13 @@ class {'self_service':
         expect(result.stdout).to match(%r{false})
         run_shell('puppet config set use_cached_catalog false', expect_failures: false)
       end
+      it 'if S0040 conditions for false are met' do
+        run_shell('echo "puppet_enterprise::system" >> /etc/puppetlabs/code/environments/production/data/common.yaml')
```

Comment:
Id move the preconditions all to the spec_helper_acceptance_local, the reason for this is the first test case at the top of this file requires the test to be "true" earlier. SO to set this up, i would add:

"puppet_enterprise::enable_system_metrics_collection: true" to the common.yaml and run puppet, in spec helper local file, and remove this echo line from here